src/client_netloop.rs

```diff
@@ -24,6 +24,13 @@ pub fn netloop(mut stream: TcpStream, gamedata: Arc<Gamedata>, pid: usize, rsbc:
                             }
                         }
                     }
+                    PktPayload::EnemyDelta(deltavec) => {
```

Comment:
we send the delta as an relative position change? by sending the absolute posisiton each time it seems like we would be less worried about desync if a packet is dropped or something